apps/ccf-eui/src/app/core/services/data-source/data-source.service.ts

```diff
@@ -21,9 +20,9 @@ export interface DelegateDataSourceOptions {
 })
 export class DelegateDataSourceService extends InjectorDelegateDataSourceService<DelegateDataSourceOptions> {
```

Comment:
We won't need a delegate data source service, we'll just use the ApiDataSourceService.